inst/startup/install_github.R

```diff
@@ -10,12 +10,12 @@ status <- tryCatch({
       packageName <- packageDirectory[length(packageDirectory)]
 
       if (!require(packageName, character.only = TRUE)) {
-        devtools::install_github(new = "$AZ_BATCH_JOB_PREP_WORKING_DIR", packageDirectory)
+        devtools::install_github(new = "$AZ_BATCH_JOB_PREP_WORKING_DIR", package)
```

Comment:
new = "$AZ_BATCH_JOB_PREP_WORKING_DIR" ? does this break the installation? I don't see new as an argument to install_github function